pyanaconda/kickstart.py

```diff
@@ -557,7 +557,8 @@ def _is_usable_disk(self, d):
         return \
             not d.format.hidden and \
             not d.protected and \
-            not (blivet.arch.is_s390() and isinstance(d, iScsiDiskDevice))
+            not (blivet.arch.is_s390() and isinstance(d, iScsiDiskDevice)) and \
```

Comment:
I think there should be a comment about what this code does:

`('local' in d.tags or getattr(d, 'ibft', False))`

And ideally also about the other conditions on the line (the s390 & iScsiDiskDevice) are there and preclude the device from being used for for the bootloder.

Reply:
Or move this to a separate method.